runtime/plugins/comment/comment.lua

```diff
@@ -19,6 +19,7 @@ ft["dart"] = "// %s"
 ft["dockerfile"] = "# %s"
 ft["elm"] = "-- %s"
 ft["fish"] = "# %s"
+ft["freest"] = "-- %s"
```

Comment:
Because the end of a selection happens to be on the new empty line it would be commented as well.

I agree that what you commented should be the correct behavior. In the last commit I added a fix for this. I added an exception to not comment the last line of a selection, if it is an empty line.

Regarding FreeST, you're right. I removed the changes. May add full language support in another PR at some point ;)